src/_layouts/post.erb

```diff
@@ -11,16 +11,22 @@ layout: default
       <%= resource.data.subtitle %>
     </p>
   </hgroup>
-  <div>
-    <small class="lowlighted">
-      <%= resource.date.strftime("%B %d, %Y") %> · <%= resource.data.author %> |
-      <a href="https://github.com/davidrunger/blog/edit/main/src/<%= resource.relative_path %>">
-        Edit this article on GitHub
-      </a>
-    </small>
+  <div class="text-sm">
+    <div class="lowlighted">
+      <%= resource.date.strftime("%B %d, %Y") %>
+      <span class="mx-1">·</span>
```

Comment:
This seemed a bit too cramped before, so I am slightly increasing the spacing by using margin rather than whitespace.